environment.yml

```
@@ -32,13 +32,10 @@ dependencies:
   - scipy
   - sphinx>=1.8.1,!=2.0.0
   - sphinx-copybutton
+  - sphinx-gallery>=0.10
   - pip
   - pip:
```

We could put a cross-reference in the conda environment file so that only one location would be the source of truth.
```suggestion
  - pip:
    - -r file:requirements/doc/doc-requirements.txt
```